sbi/simulators/user-input-checks.md

```diff
@@ -0,0 +1,62 @@
+# User input checks
```

Comment:
I'd propose to move this into `docs/`

Reply:
Yes, this isnt meant for the user yet, it was more a plan for me to have an overview of all the checks. I will refine this to a README for the user and move it to the right place then.